src/utils/odoc_utils.ml

```diff
@@ -79,4 +79,44 @@ module Tree = Tree
 module Forest = Tree.Forest
 module Json = Json
 
+module Io_utils = struct
+  let _with_resource res ~close f =
```

Comment:
Is this starting with an underscore to indicate it's not for use outside of the module?

Reply:
Exactly. This avoids writing a signature to abstract this one function while the others are trivial.

Reply:
I just added some documentation to the module to make it easier to read without an interface.